schema/drone_schema.py

```diff
@@ -0,0 +1,5 @@
+from schema.base_schema import BaseSchema
+
+
+class DroneSchema(BaseSchema):
```

Comment:
to avoid a misunderstanding - the beginning of a schema should be started with the name of a model and then add "...Admin", "...Internal", "...Update", "...ToDrone" etc.